.github/workflows/ruby-qltest.yml

```diff
@@ -5,8 +5,11 @@ on:
     paths:
       - "ruby/**"
       - "shared/**"
-      - .github/workflows/ruby-build.yml
+      - .github/workflows/ruby-qltest.yml
       - .github/actions/fetch-codeql/action.yml
+      - .github/actions/cache-query-compilation/action.yml
+      - .github/actions/reasonable-memory/action.yml
+      - csharp/actions/create-extractor-pack/action.yml
```

Comment:
this is amusing -- it probably should be moved to .github/actions -- the idea that ruby is using a csharp action is pretty confusing.